akka-bbb-apps/run-dev.sh

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
+sudo service bbb-apps-akka stop
+
 rm -rf src/main/resources
 cp -R src/universal/conf src/main/resources
-exec sbt update run
+
+#Set correct sharedSecret and bbbWebAPI
+sudo sed -i "s/sharedSecret = \"changeme\"/sharedSecret = \"$(sudo bbb-conf --salt | grep Secret: | cut -d ' ' -f 6)\"/g" src/main/resources/application.conf
```

Comment:
These two could be also handled by a conf override in /etc/bigbluebutton.
If you use `sed` here, it will result in the conf file occasionally being committed incorrectly or even included in PRs.
Just an idea

Reply:
Maybe it worth to apply the configs to the project config because we use to run Akka directly from the IDE, from outside of the Docker. And it will need the correct `sharedSecret`.

It will not be committed incorrectly once the destination folder `src/main/resources/` are being ignored already.

![image](https://user-images.githubusercontent.com/5660191/161117639-634df02a-e8bf-402e-8145-22cb57cd5f19.png)